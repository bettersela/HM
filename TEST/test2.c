// hex test
void main() {
  n = 0x100;
  print n;//  | PRINT Expr { Prim1("printi", $2) }
  println;//  | PRINTLN  { Prim1("printc", nl) }
}
