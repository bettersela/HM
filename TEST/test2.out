15 1 24 19 0 7 25 0 0 0 256 12 15 -1 0 0 11 22 15 -1 0 10 23 15 -1 15 0 21 -1